2-mandatory/2-currency-code-lookup.js

```diff
@@ -19,8 +19,14 @@ const COUNTRY_CURRENCY_CODES = [
 
 function createLookup(countryCurrencyCodes) {
   // write code here
+  const currencyCode = {};
+  for (let item of countryCurrencyCodes) {
+    currencyCode[item[0]] = item[1];
+  }
+  return currencyCode;
 }
```

Comment:
Comment: Well done!

Consider: You could also use .forEach https://www.w3schools.com/jsref/jsref_foreach.asp 
function createLookup(countryCurrencyCodes) {
  // write code here
  const lookup = {};
  countryCurrencyCodes.forEach(([country, currency]) => {
    lookup[country] = currency;
  });
  return lookup;
}

const lookup = createLookup(COUNTRY_CURRENCY_CODES);
console.log(lookup);